nsv13/code/game/objects/items/custom_guns.dm

```diff
@@ -139,3 +139,21 @@
 	ammo_type = /obj/item/ammo_casing/shotgun/buckshot
 	max_ammo = 6
 	multiple_sprites = 1
+
+/obj/item/gun/ballistic/rifle/boltaction/pdc
+	name = "\improper Point Defence Rifle"
+	desc = "A rifle made to fire PDC rounds."
+	icon = 'nsv13/icons/obj/guns/projectile.dmi'
+	icon_state = "pdcrifle"
+	item_state = "pdcrifle"
+	lefthand_file = 'nsv13/icons/mob/inhands/weapons/guns_lefthand.dmi'
+	righthand_file = 'nsv13/icons/mob/inhands/weapons/guns_righthand.dmi'
+	slot_flags = ITEM_SLOT_BACK
+	mag_type = /obj/item/ammo_box/magazine/nsv/pdc/rifle
+	can_bayonet = TRUE
+	knife_x_offset = 27
+	knife_y_offset = 13
+	w_class = 3 // It's a syndicate weapon so weight is reduced to allow it to be hidden in backpacks
```

Comment:
Debatable if this should be the case, it's a pretty powerful weapon.

Reply:
Please also use actual defines for the weight class rather than just numbers.

Reply:
If the weight class was higher then the rifle would be a downgrade to the revolver in almost every scenario, considering the revolver does slightly more damage, has ammo that can be printed, has a 6(?) round magazine and can be stored in the backpack. That's the only reason I lowered the weight class, but if you still think I should change it then I'll go ahead and do that.

Reply:
> If the weight class was higher then the rifle would be a downgrade to the revolver in almost every scenario, considering the revolver does slightly more damage, has ammo that can be printed, has a 6(?) round magazine and can be stored in the backpack. That's the only reason I lowered the weight class, but if you still think I should change it then I'll go ahead and do that.

It should still use a define like Karmic said either way, but that makes sense then.
Please look at my other comments as well.